health-services/household/pom.xml

```diff
@@ -45,12 +45,12 @@
         <dependency>
             <groupId>org.egov.common</groupId>
             <artifactId>health-services-common</artifactId>
-            <version>1.0.18-SNAPSHOT</version>
+            <version>1.0.19-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.egov.common</groupId>
             <artifactId>health-services-models</artifactId>
-            <version>1.0.20-SNAPSHOT</version>
+            <version>1.0.24-dev-snapshot</version>
```

Comment:
please sit with Yashita and get her enum changes merged to the same version of the library

Reply:
She has added the same changes in data model package for apk